desktop-exporter/app/components/sidebar-view/sidebar.tsx

```diff
@@ -13,36 +26,73 @@ type SidebarProps = {
 
 export function Sidebar(props: SidebarProps) {
   let sidebarColour = useColorModeValue("gray.50", "gray.700");
+  let iconColour = useColorModeValue("white", "pink.900");
+  let { colorMode, toggleColorMode } = useColorMode();
   let { isFullWidth, toggle, traceSummaries } = props;
 
-  let sidebarWidth = "70px";
-  let buttonIcon = <ArrowRightIcon />;
-  let traceList = <></>;
-
   if (isFullWidth) {
-    sidebarWidth = "350px";
-    buttonIcon = <ArrowLeftIcon />;
-    traceList = <TraceList traceSummaries={traceSummaries} />;
+    return (
+      <Flex
```

Comment:
There's enough here that it might be good to break this into multiple components: `CollapsedSideBar` and `ExpandedSideBar`

Reply:
Once I pulled out the buttons, as per your next suggestion, there isn't really much to the sidebar, so I'm going to leave it as a single component if that's ok.